src/Cdn/Cdn/help/New-AzCdnDeliveryPolicy.md

```diff
@@ -0,0 +1,92 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Cdn.dll-Help.xml
+Module Name: Az.Cdn
+online version:
```

Comment:
@hytao for each of the new markdown files you're adding, there should be a URL added for the `online version` property in the header of the file. It would be in the form of `https://docs.microsoft.com/en-us/powershell/module/az.cdn/{{cmdlet-name}}`

See [this file](https://raw.githubusercontent.com/Azure/azure-powershell/master/src/Cdn/Cdn/help/Confirm-AzCdnEndpointProbeURL.md) for an example of the URL at the top of the file.